privacysandbox/privacysandbox.go

```diff
@@ -0,0 +1,191 @@
+package privacysandbox
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/prebid/openrtb/v20/openrtb2"
+)
+
+const (
+	topicSegmentSeparator = ","
+	topicMaxLength        = 10
+)
```

Comment:
Given that these consts are only used once, do they need to be consts at all?

Reply:
done